core/src/main/java/nl/nn/adapterframework/http/HttpSenderBase.java

```diff
@@ -534,18 +534,6 @@ public void setCharSet(String string) {
 		charSet = string;
 	}
 
-	@Deprecated
-	@ConfigurationWarning("Please use attribute username instead")
-	public void setUserName(String username) {
-		setUsername(username);
-	}
-
-	@Deprecated
-	@ConfigurationWarning("Please use \"proxyUsername\" instead")
-	public void setProxyUserName(String string) {
-		setProxyUsername(string);
-	}
-
 	@Deprecated
```

Comment:
I probably missed something, why is this attribute deprecated but not removed?

Reply:
Because it belongs to other properties which Niels did not set on the list to remove. It belongs to `setCertificate`, see the code below.